torch/nn/modules/batchnorm.py

```diff
@@ -659,12 +658,6 @@ def _check_non_zero_input_channels(self, input):
                 "SyncBatchNorm number of input channels should be non-zero"
             )
 
-    def _specify_ddp_gpu_num(self, gpu_size):
```

Comment:
was the callee removed from fairscale repo as well?

Reply:
Yes, the relevant deletions are shown in the internal diffs, as it's not in PyTorch repo.